providers/cloudflare.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "cloudflare",
+  "services": [
+    {
+      "id": "challenges",
+      "baseUrl": "https://challenges.cloudflare.com/"
+    }
+  ],
+  "defaultService": "challenges",
```

Comment:
question: Is it necessary to define default service? Assuming we have Cloudflare for all of their APIs, I find it unlikely most maps will use the challenges service.

Reply:
Good point. The default service should not be Turnstile challenges service address.